utils.go

```diff
@@ -74,6 +75,11 @@ func (r *rawModeHandler) Exit() error {
 	return err
 }
 
+func clearScreen(w io.Writer) error {
+	_, err := w.Write([]byte("\x1b[H\x1b[J"))
```

Comment:
Maybe at some point named constant etc for ansi stuff could be nice for readability?

Reply:
Yeah, maybe. What I've found in the past is that the composition rules are tricky. [ircdog](https://github.com/ergochat/ircdog) used to use [mgutz/ansi](https://github.com/mgutz/ansi) --- I ended up removing it and handcrafting my own ANSI codes because I found that it obscured more than it clarified.

Reply:
Aha i see, yeah i actually ran into something similar when trying to general ANSI-aware line wrapper for fq :)